packages/console/src/components/Tooltip/index.tsx

```diff
@@ -11,24 +11,42 @@ type Props = {
   anchorRef: RefObject<Element>;
   className?: string;
   isKeepOpen?: boolean;
+  horizontalAlign?: HorizontalAlignment;
 };
 
-const Tooltip = ({ content, anchorRef, className, isKeepOpen = false }: Props) => {
+const getHorizontalOffset = (alignment: HorizontalAlignment): number => {
+  switch (alignment) {
+    case 'start':
+      return -32;
```

Comment:
nit: Just wondering where does the `32` come from?

Reply:
i admit it's a magic number (calculated by the design file, basically `unit(8)`). can we optimize later?

Reply:
LOG-3642